app/views/voice_in_trunk_groups/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="panel panel-default">
   <div class="panel-heading">
-    <%= link_to 'My Trunks', trunks_path %> >
-    Create <%= resource.configuration_type_human %> Trunk
+    <%= link_to 'My Voice In Trunk Groups', voice_in_trunk_groups_path %> >
```

Comment:
Please put Voice In to quotes to make it readable
Example: My "Voice In" Trunk Groups and apply this for all the trunks (in and out) displaying in all views